cmd/experimental/kjobctl/pkg/cmd/create/create.go

```diff
@@ -0,0 +1,269 @@
+	cmd.Flags().StringVar(&o.UserSpecifiedCommand, commandFlagName, "",
+		"Command which is associated with the resource.")
+	cmd.Flags().Int32Var(&o.UserSpecifiedParallelism, parallelismFlagName, 0,
+		"Parallelism specifies the maximum desired number of pods the job should run at any given time.")
+	cmd.Flags().Int32Var(&o.UserSpecifiedCompletions, completionsFlagName, 0,
+		"Completions specifies the desired number of successfully finished pods the.")
+	cmd.Flags().StringToStringVar(&o.UserSpecifiedRequest, requestFlagName, nil,
+		"Request is a set of (resource name, quantity) pairs.")
+	cmd.Flags().StringVar(&o.LocalQueue, localQueueFlagName, "",
+		"Kueue localqueue name which is associated with the resource.")
+
+	util.AddDryRunFlag(cmd)
+
+	_ = cmd.MarkFlagRequired(profileFlagName)
+
+	cobra.CheckErr(cmd.RegisterFlagCompletionFunc(localQueueFlagName, completion.LocalQueueNameFunc(clientGetter)))
+
+	return cmd
+}
+
+func (o *CreateOptions) Complete(clientGetter util.ClientGetter, cmd *cobra.Command, args []string) error {
+	switch args[0] {
+	case "job":
+		o.ModeName = v1alpha1.JobMode
+	case "interactive":
+		o.ModeName = v1alpha1.InteractiveMode
+	default:
+		return invalidApplicationProfileModeErr
+	}
+
+	var err error
+
+	o.Namespace, _, err = clientGetter.ToRawKubeConfigLoader().Namespace()
+	if err != nil {
+		return err
+	}
+
+	if o.UserSpecifiedCommand != "" {
+		o.Command = strings.Fields(o.UserSpecifiedCommand)
+	}
+
+	if flag := cmd.Flags().Lookup(parallelismFlagName); flag.Changed {
+		o.Parallelism = ptr.To(o.UserSpecifiedParallelism)
+	}
+
+	if flag := cmd.Flags().Lookup(completionsFlagName); flag.Changed {
+		o.Completions = ptr.To(o.UserSpecifiedCompletions)
+	}
+
+	if len(o.UserSpecifiedRequest) > 0 {
+		o.Requests = make(corev1.ResourceList)
+		for key, value := range o.UserSpecifiedRequest {
+			quantity, err := apiresource.ParseQuantity(value)
+			if err != nil {
+				return err
+			}
+			o.Requests[corev1.ResourceName(key)] = quantity
+		}
+	}
+
+	o.DryRunStrategy, err = util.GetDryRunStrategy(cmd)
+	if err != nil {
+		return err
+	}
+
+	err = util.PrintFlagsWithDryRunStrategy(o.PrintFlags, o.DryRunStrategy)
+	if err != nil {
+		return err
+	}
+
+	printer, err := o.PrintFlags.ToPrinter()
+	if err != nil {
+		return err
+	}
+
+	o.PrintObj = printer.PrintObj
+
+	return nil
+}
+
+func (o *CreateOptions) Run(ctx context.Context, clientGetter util.ClientGetter) error {
+	obj, err := builder.NewBuilder(clientGetter).
+		WithNamespace(o.Namespace).
+		WithProfileName(o.ProfileName).
+		WithModeName(o.ModeName).
+		WithCommand(o.Command).
+		WithParallelism(o.Parallelism).
+		WithCompletions(o.Completions).
+		WithRequests(o.Requests).
+		WithLocalQueue(o.LocalQueue).
+		Do(ctx)
+	if err != nil {
+		return err
+	}
+
+	if o.DryRunStrategy != util.DryRunClient {
+		obj, err = o.createObject(ctx, clientGetter, obj)
+		if err != nil {
+			return err
+		}
+	}
+
+	return o.PrintObj(obj, o.Out)
```

Comment:
Just a note for now, the `Interactive` mode should keep the stdin stdout  streams open.